packages/devtools_extensions/README.md

```diff
@@ -2,10 +2,13 @@
 
 Extend Dart & Flutter's developer tool suite,
 [Dart DevTools](https://docs.flutter.dev/tools/devtools/overview), with a custom tool for
-your package. DevTools' extension framework allows you to build a tool for your Dart package
-that can leverage existing frameworks and utilities from DevTools (VM service connection, theming,
-shared widgets, utilities, etc.). When an app is connected to DevTools that depends on your
-package, your extension will show up in its own DevTools tab:
+your package.
+
+DevTools' extension framework allows you to add your custom tab to the DevTools application
+by creating a Dart package
+that leverage frameworks and utilities from DevTools (like VM service connection, theming,
+shared widgets, utilities, etc.). When an app, that depends on your
```

Comment:
What about updating this to wording that is similar to the medium article:

With DevTools extensions framework, you can easily build developer tooling that is tightly integrated with the existing DevTools tooling suite. Extensions are built using Flutter web and leverage existing frameworks and utilities from DevTools to simplify the developer tool authoring experience. 

Your extension can ship as its own standalone Dart package or as part of an existing Dart package. When an app that depends on your package is connected to DevTools, your extension will show up in its own DevTools tab.

Reply:
Extenstions in simple words are about adding a custom tab to DevTools. 
I want the first sentence to clearly say this. And then second paragraph to explain details about how it is done. 

So how about adding the sentence 'DevTools extension framework allows you to add your custom tab to the DevTools application' on the very top?